doc/hmc_doc.md

```diff
@@ -0,0 +1,15 @@
+HMC Documentation Page
+
+fix hmc command
+
+Syntax
+
```

Comment:
Description before examples? Or no? I guess having syntax very close to examples is nice...